web/src/scenes/actions/NewAction/Form/FieldInput/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { InputHTMLAttributes, memo } from "react";
```

Comment:
Same as [here](https://github.com/worldcoin/developer-portal/pull/248#discussion_r1157054805). We have a common input component. We can refactor it, but this will cause multiple changes across the app and the PR diff will be massive